pytest_testrail/conftest.py

```diff
@@ -97,7 +100,8 @@ def pytest_configure(config):
                 tr_name=config_manager.getoption('tr-testrun-name', 'name', 'TESTRUN'),
                 run_id=config.getoption('--tr-run-id'),
                 plan_id=config.getoption('--tr-plan-id'),
-                version=config.getoption('--tr-version')
+                version=config.getoption('--tr-version'),
+                close_on_complete=config.getoption('--close-on-complete')
```

Comment:
command line options, should be prefixed with 'tr'

Reply:
@dubner  fixed, thanks

Reply:
@dubner please review changes.